test/misc-tests.js

```diff
@@ -1566,4 +1566,44 @@ describe("miscellaneous tests", () => {
       import("./fixture/source/source-mapping-url-string.mjs")
     )
   })
+
+    // Test for https://github.com/standard-things/esm/issues/591
+  describe("Workers support", () => {
+    before(function ()  {
+      if (!process.version.startsWith("v10.") && !process.version.startsWith("v11.")) {
+        this.skip()
+      }
```

Comment:
At the top of the misc-test.js file you'll see other `canDoThisOrThat` vars. You can add one for
```
const canTestWorkerThreads = SemVer.satisfies(process.version, ">10.5.0")
```